golang/cosmos/app/app.go

```diff
@@ -773,8 +789,7 @@ func NewAgoricApp(
 
 func upgrade10Handler(app *GaiaApp, targetUpgrade string) func(sdk.Context, upgradetypes.Plan, module.VersionMap) (module.VersionMap, error) {
 	return func(ctx sdk.Context, plan upgradetypes.Plan, fromVm module.VersionMap) (module.VersionMap, error) {
-		// TODO: In next upgrade handler (bulldozer), run:
-		// app.VstorageKeeper.MigrateNoDataPlaceholders(ctx)
+		app.VstorageKeeper.MigrateNoDataPlaceholders(ctx)
```

Comment:
Maybe add a comment to mention this should only be run in Bulldozer, and not in future upgrades (aka do not include in upgrade11).

Also we'll need a staging environment equivalent I suppose?

Reply:
Added a note, though I think it's implicit that the contents of a specific upgrade handler are just for that specific upgrade.

The staging upgrade should be handled by the `upgradeNameTest` handler installed on line 749.